packages/react-scripts/package.json

```diff
@@ -30,8 +30,8 @@
   "dependencies": {
     "@babel/core": "7.4.3",
     "@svgr/webpack": "4.1.0",
-    "@typescript-eslint/eslint-plugin": "1.6.0",
-    "@typescript-eslint/parser": "1.6.0",
+    "@typescript-eslint/eslint-plugin": "1.7.1-alpha.1",
```

Comment:
I don't think we want to ship an alpha version just to remove some warnings. Can we wait until this is stable and upgrade it after the 3.0 release?

Reply:
We can yes. This is a commit after their `1.7.0` release as they release automatically on commit to master.